### PR TITLE
Add aoe2netwrapper package

### PR DESCRIPTION
**Pull request by fsoubelet**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/aoe2netwrapper/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (dopplershift) [inline recipes/aoe2netwrapper/LICENSE:1]: This is already included in the PyPI tarball used to build the package, so it should not be included here. Please go ahead and remove.

File: recipes/aoe2netwrapper/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - aoe2netwrapper
+    - aoe2netwrapper.models
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/fsoubelet/AoE2NetAPIWrapper
+  summary: My Python wrapper for the aoe2.net data API
```
Comment (dopplershift) [inline recipes/aoe2netwrapper/meta.yaml:40]:
```suggestion
  summary: Python wrapper for the aoe2.net data API
```
?

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a couple minor tweaks and this is good to go.

**Comment by fsoubelet** [comment]
Thanks for providing pointers @dopplershift, I've made the changes 🙂

**Review by dopplershift — APPROVED** [review]

**Comment by dopplershift** [comment]
I'm ignoring the Windows failure because it's from an unrelated recipe that was merged.

**Pull request merged**